addons/payment/views/payment_templates.xml

```diff
@@ -56,14 +56,79 @@
+                    <t t-foreach="tokens" t-as="token">
+                        <t t-set="provider" t-value="token.provider_id"/>
+                        <div name="o_payment_option_card" class="card-body o_payment_option_card">
+                            <label>
+                                <!-- === Radio button === -->
+                                <input name="o_payment_radio"
+                                       type="radio"
+                                       t-att-checked="token.id == default_payment_option_id"
+                                       t-att-data-payment-option-id="token.id"
+                                       t-att-data-provider="token.provider_code"
+                                       data-payment-option-type="token"/>
+                                <!-- === Token name === -->
+                                <span class="payment_option_name" t-esc="token.display_name"/>
+                                <!-- === "V" check mark === -->
+                                <t t-call="payment.verified_token_checkmark"/>
+                                <!-- === "Fees" badge === -->
+                                <span t-if="fees_by_provider.get(token.provider_id)"
+                                      class="badge rounded-pill text-bg-secondary ms-1">
+                                        + <t t-esc="fees_by_provider.get(token.provider_id)"
+                                             t-options="{'widget': 'monetary', 'display_currency': currency}"/>
+                                        Fees
+                                </span>
+                                <!-- === "Test token" badge === -->
+                                <t t-call="payment.test_token_badge"/>
+                                <!-- === "Unpublished" badge === -->
+                                <span t-if="not token.provider_id.is_published" class="badge rounded-pill text-bg-danger ms-1">
+                                    Unpublished
+                                </span>
+                                <div class="ms-4">
+                                    <span class="fst-italic small text-muted">
+                                        Managed by
+                                        <!-- It would not work with custom payment provider e.g. 
+                                          "Wire transfer" as it would just show "custom", but
+                                          for now we don't tokenize custom providers, so it's
+                                          not a problem -->
+                                        <span class="token_payment_provider_code"
+                                              t-esc="dict(provider._fields['code']._description_selection(provider.env))[provider.code]"/>
+                                        - created on 
+                                        <span class="token_payment_creation_date" t-esc="token.create_date.date()"/>
```

Comment:
```suggestion
                                        <span class="token_payment_creation_date"
                                                    t-esc="token.create_date.date()"/>  
```

Reply:
Also, swap `class` and `t-esc`; the reader wants to know what is printed before how it's stylized.